css/layout.css

```diff
@@ -25,9 +27,8 @@
 }
 
 .footer {
-    height: 80px;
     background-color: var(--primary-light);
-    padding: 0 var(--size-3);
+    padding: var(--size-3) 20px;
```

Comment:
I'd recommend to use a size variable here -- if `--size-3` or `--size-4` does not look right, consider using `calc(var(--size-3) * 1.5)`;

Using size variables tends to result in cleaner looking UIs.